utils/coreutils/utils.go

```diff
@@ -198,6 +199,27 @@ func GetWorkingDirectory() (string, error) {
 	return currentDir, nil
 }
 
+// IsTerminal checks whether stdout is a terminal.
+func IsTerminal() bool {
+	return terminal.IsTerminal(int(os.Stdout.Fd()))
+}
+
+var coloredOutput *bool
+
+// IsColoredOutput returns whether the output can be colored.
```

Comment:
IsColoredOutput returns whether the output can be colored.
-->
IsColoredOutput returns true if the output can be colored.